db/db_test.cc

```diff
@@ -178,6 +181,9 @@ TEST_F(DBTest, RequestIdPlumbingTest) {
     // Verify the request_id was propagated to the file system
     ASSERT_NE(captured_request_id_dbg, nullptr);
     ASSERT_EQ(*captured_request_id_dbg, test_request_id);
+
+    ASSERT_NE(dbgCopy.request_id, nullptr);
+    ASSERT_EQ(*dbgCopy.request_id, test_request_id);
```

Comment:
I suggest also adding a check that `dbgCopy.request_id != dbg->request_id`

Reply:
makes sense, updated